render/vulkan/vulkan.c

```diff
@@ -33,6 +33,81 @@ static const char *find_extensions(const VkExtensionProperties *avail,
 	return NULL;
 }
 
+static char **get_validation_layers(uint32_t *layer_count) {
+	// TODO request layer via env var
+	static const char *layers[] = {
+		"VK_LAYER_KHRONOS_validation",
+		// "VK_LAYER_RENDERDOC_Capture",
+		// "VK_LAYER_live_introspection",
+	};
+
+	static const size_t layers_len = sizeof(layers) / sizeof(layers[0]);
+
+	VkLayerProperties *layer_props = NULL;
+	char **found_layers = NULL;
+	uint32_t found_layers_count = 0;
+
+	uint32_t count;
+	vkEnumerateInstanceLayerProperties(&count, NULL);
+
+	if (count == 0) {
```

Comment:
Could also check that the return value from the `vkEnumerateInstanceLayerProperties` above is `VK_SUCCESS`.